src/Level/index.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import {default as PT} from 'prop-types'
+import cx from 'classnames'
+
+const Level = ({ variant }) => {
+  const styles = cx('ola_level', `is-${variant}`)
```

Comment:
`variant` is not a required property, so the `is-${variant}` classname shoud be added only if variant is defined.